src/Card/__tests__/Card.tests.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import Card from '../Card'
+
+describe('<Card />', () => {
+    it('renders CardProps', () => {
+        const { container, getByText } = render(
+            <Card
+                title='My Card'
+                description='This is a card description.'
+                content={<span>Span Content</span>}
+                disabled={false}
+                onClick={() => undefined}
+            />);
+
+        getByText('My Card');
+        getByText('This is a card description.');
+        getByText('Span Content');
+        expect(container.querySelector('button')).toBeInTheDocument();
+    });
+
+    it('calls onClick when clicked and disabled is false', async () => {
+        const click = jest.fn();
+        const { container } = render(
+            <Card
+                title='My Card'
+                description='This is a card description.'
+                content={<span>Span Content</span>}
+                disabled={false}
+                onClick={click}
+            />);
+
+        const button = container.querySelector('button');
+        await waitFor(() => fireEvent.click(button));
+
+        expect(click).toHaveBeenCalled();
+    });
+
+    it('does not call onClick when disabled is true', async () => {
+        const click = jest.fn();
+        const { container } = render(
+            <Card
+                title='My Card'
+                description='This is a card description.'
+                content={<span>Span Content</span>}
+                disabled={true}
+                onClick={click}
+            />);
+
+        const button = container.querySelector('button');
+        await waitFor(() => fireEvent.click(button));
+
+        expect(click).not.toHaveBeenCalled();
+        expect(button).toBeDisabled();
+    });
```

Comment:
nit: Still should have an additional test to show that a "custom" topRightComponent button is rendered (or not).